### ISSUE TITLE
TestGenerator: unmatched data error, low value is not reached from the least timeframe [$80]

### ISSUE THREAD

**Issue by kenorb** [FX31337/FX-BT-Scripts#71]
Error happening during backtesting:

M5:
```
TestGenerator: unmatched data error (low value 1.08391 at 2015.07.20 09:15 is not reached from the least timeframe, low price 1.08474 mismatches)
TestGenerator: unmatched data error (volume limit 590 at 2015.07.20 09:30 exceeded)
```

M15:

```
TestGenerator: unmatched data error (low value 1.08294 at 2015.07.20 09:15 is not reached from the least timeframe, low price 1.08474 mismatches)
TestGenerator: unmatched data error (low value 1.08294 at 2015.07.20 09:15 is not reached from the least timeframe, low price 1.08474 mismatches)
```

M30
```
TestGenerator: unmatched data error (low value 1.08294 at 2015.07.20 09:00 is not reached from the least timeframe, low price 1.08391 mismatches)
TestGenerator: unmatched data error (low value 1.08294 at 2015.07.20 09:00 is not reached from the least timeframe, low price 1.08391 mismatches)
```

Tested data from [2015](https://github.com/FX31337/FX-BT-Data-EURUSD-DS/releases/tag/2015).

--

Testing (using [VM](https://github.com/EA31337/FX-MT-VM)):

1. Close VM repo.
1. Optionally provision VM by: `vagrant up && vagrant ssh` (or use local Linux/OSX shell)
1. Run the test using M30 period, e.g. from the command line:

        ./scripts/run_backtest.sh -t -v -x -I TestPeriod=M30,TestModel=0 -e MACD -p EURUSD -d 2000 -s 10 -y 2015 -m 7 -D 5 -b DS -M 4.0.0.971
1. Clone [scripts](https://github.com/FX31337/FX-BT-Scripts) repo.
1. Clone/download some [CSV files](https://github.com/FX31337/FX-BT-Data-EURUSD-DS/tree/EURUSD-2015).
1. Run: `./convert_csv_to_mt.py -v -i all.csv -s EURUSD -p 20 -S default -t M1 -f fxt4`.
1. Copy FXT files to platform dir at: `tester/history/`.
1. To read FXT format, use: `./convert_mt_to_csv.py -i EURUSD1_0.fxt -f fxt4 | more` instead.

Fix should be in [`convert_csv_to_mt.py`](https://github.com/FX31337/FX-BT-Scripts/blob/master/convert_csv_to_mt.py) file.

Similar issue: #70

--

The solution to this issue is to fix the converter script, so the above errors won't show for all timeframes, so the backtest test can run successfully.

--

Es. 6-8h

**Comment by nseam** [comment]
Finally replicated. yuppii

**Comment by kenorb** [comment]
Fixed in #72 & #75
